doxyfile_options

```diff
@@ -845,7 +845,7 @@ EXCLUDE_PATTERNS       = */tools/* \
                          */features/mbedtls/* \
                          */features/storage/* \
                          */features/unsupported/* \
-                         */features/FEATURE_LWIP/* \
+                         */features/lwipstack/* \
```

Comment:
It's probably more straightforward just to be `/features/lwip`, I think. Does the "stack" really add anything in this context?

`LWIPStack` versus `LWIPInterface` objects kind of happened because they were in a single namespace, and reflected the underlying `NetworkStack` and `NetworkInterface`.  But in the filing system like this, I think just `lwip` is fine. Maybe at some point `lwip` it might end up in a `networkstacks` folder, if there were that many.

Reply:
It's possible there should be a general tidy-up to add a top-level `features/networking` or something too. But I think that's out of scope for this. Anyway, for now don't try to express something in your leafname that would be better expressed in a directory structure.

Reply:
Within the `lwipstack` there is already folder `lwip` which is the ... stack.. :D

Would `lwip/lwip` look terrible?  Currently the stack is in `lwipstack/lwip`